src/main/java/reposense/system/CommandRunner.java

```diff
@@ -56,7 +56,7 @@ public static String runCommand(Path path, String command) {
         }
     }
 
-    private static boolean isWindows() {
+    public static boolean isWindows() {
```

Comment:
Probably can shift this method to a `SystemUtil` class instead.
We also can probably also remove the `CommandRunner` class, and shift the `runCommand` method there to `runSystemCommand` instead, but not in this PR though.